### PR TITLE
teams/docs: a vote for hfxsd as the reviewer

### PR DESCRIPTION
**Pull request by lilin90**
As stated in the title.

Hi @pingcap/tidb-maintainers, PTAL.

/hold

### CODE REVIEW

**Review by TomShawn — APPROVED** [review]
LGTM

File: votes/0593-hfxsd-as-docs-reviewer.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+I (@lilin90) hereby nominate @hfxsd as Docs Reviewer and call for a vote.
+
+## Deadline
+
+The vote will be open for at least 3 days unless there is an objection or not enough votes.
+
+## Scope
+
+* Team Docs
```
Comment (qiancai) [inline votes/0593-hfxsd-as-docs-reviewer.md:27]: Docs Team?

**Review by qiancai — APPROVED** [review]

**Comment by qiancai** [comment]
LGTM

**Comment by qiancai** [comment]
+ Approval  @hfxsd has rich experience in technical writing and is a good teacher for all members of the Docs team. Congratulations!

**Comment by TomShawn** [comment]
+ Approval Happy to see @hfxsd is finally nominated as the Reviewer! Congratulation!🎉

**Review by disksing — APPROVED** [review]

File: votes/0593-hfxsd-as-docs-reviewer.md (hunk 1 above)
Comment (lilin90) [reply votes/0593-hfxsd-as-docs-reviewer.md:27]: @qiancai I also wanted to use "Docs Team", but in the existing files in [votes](https://github.com/pingcap/community/tree/master/votes), the "Team xxx" style is used. Is this on purpose? @tisonkun

Comment (tisonkun) [reply votes/0593-hfxsd-as-docs-reviewer.md:27]: No. It's a suggestion. You may change if you think "Docs Team" better.

Comment (lilin90) [reply votes/0593-hfxsd-as-docs-reviewer.md:27]: > No. It's a suggestion. You may change if you think "Docs Team" better.

@tisonkun Got it. Or we can use "Team: xxx".

**Review by tangenta — APPROVED** [review]

**Comment by lilin90** [comment]
This voting has met the requirement and I'm going to merge this PR. If you have any concerns, please let me know. Thanks!

**Pull request merged**